### PR TITLE
Fix Websocket APIs are not working when correlation logs are enabled

### PR DESCRIPTION
**Pull request by YasasRangika**
## Purpose
To fix Websocket APIs are not working when correlation logs are enabled

## Goals
fixes: https://github.com/wso2/product-apim/issues/11836

## Approach
Added validation to check whether the headers are not available.

Git actions: https://github.com/YasasRangika/carbon-apimgt/actions/runs/1529688729

### CODE REVIEW

**Review by tharikaGitHub — APPROVED** [review]

**Review by tharindu1st — APPROVED** [review]

**Comment by tharindu1st** [comment]
need a unit test to cover the scenarios

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/test/java/org/wso2/carbon/apimgt/gateway/handlers/LogsHandlerTestCase.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package org.wso2.carbon.apimgt.gateway.handlers;
```
Comment (mushthaq33) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/test/java/org/wso2/carbon/apimgt/gateway/handlers/LogsHandlerTestCase.java:1]: License header?

Comment (YasasRangika) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/test/java/org/wso2/carbon/apimgt/gateway/handlers/LogsHandlerTestCase.java:19]: Added with https://github.com/wso2/carbon-apimgt/pull/10939/commits/cd3f487bee1fa5534a3e911c2ea7b2bff4a055de

**Review by tharindu1st — APPROVED** [review]

**Review by 1akshitha — APPROVED** [review]

**Pull request merged**